agent/lib/kontena/launchers/ipam_plugin.rb

```diff
@@ -40,6 +40,14 @@ def image_exists?
       @image_pulled
     end
 
+    def running?
```

Comment:
Just pushed some improvement for this. Now it emits the start event only when ipam is running. And it should not be running without etcd. :)